README.md

```diff
@@ -1,3 +1,11 @@
+**The following project components have been deprecated and moved under [Node Tuning Operator](https://github.com/openshift/cluster-node-tuning-operator):**
+- Performance Addon Operator, now [Performance Addon controller](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_addon.md), a day-0 controller deployed with Node Tuning Operator.
+- [Performance profile creator](https://github.com/openshift/cluster-node-tuning-operator/blob/master/cmd/performance-profile-creator/README.md).
+- Render mode.
```

Comment:
I'd add a reference that all of this is valid for >= 4.11 and versions 4.10 are still supported in bugfix mode

Reply:
other than this, LGTM